qa/L0_config_json/test.sh

```diff
@@ -373,6 +373,52 @@ fi
 kill $SERVER_PID
 wait $SERVER_PID
 
+# Test max_priority_level
+TRIAL=max_priority_level
+
+rm -fr models && mkdir models
+mkdir -p models/max_priority_level/1 && cp max_priority_level.pbtxt models/max_priority_level/config.pbtxt
+
+run_server
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "\n***\n*** Failed to start $SERVER\n***"
+    cat $SERVER_LOG
+    exit 1
+fi
+
+set +e
+code=`curl -s -w %{http_code} -o ./$TRIAL.out localhost:8000/v2/models/max_priority_level/config`
```

Comment:
Generally, if you name things with `.log` suffix, it will be more likely to get automatically wrapped up in job artifacts for easier debugging in CI. So I'd prefer `${TRIAL}.log` or similar, instead of `.out`

Reply:
Separate PR is fine, and sure if it's all throughout the file a find/replace would be good.